Form/Type/DynamicFormType.php

```diff
@@ -149,6 +157,19 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             'mapped' => false,
         ]);
 
+        if ($this->honeyPotField) {
+            $builder->add(
+                str_replace(' ', '_', strtolower($this->honeyPotField)),
+                EmailType::class,
+                [
+                    'label' => $this->honeyPotField,
+                    'mapped' => false,
+                    'block_name' => 'honeypot',
```

Comment:
`block _honeypot_row` didn't work? then this can be removed

Reply:
it does work but with prefix of the form so _dynamic_honeypot_row

Reply:
okay